### DISCUSSION TITLE
After updating from Filament v1.9.23 to v1.9.24, nothing is drawn

### DISCUSSION THREAD

**Discussion by cx20** [google/filament#3927] [Q&A] [answered]
I changed the version of the library used in [gltf-test](https://github.com/cx20/gltf-test) from Filament `v1.9.23` to `v1.9.24`.
However, nothing seems to show up.

[Filament v1.9.23 + Duck.gltf result](https://rawcdn.githack.com/cx20/gltf-test/6ccc13051e12cfe97b8b93c72cf13fbec6b0236f/examples/filament/index.html?category=sampleModels&model=Duck&scale=1&type=glTF): OK

[Filament v1.9.24 + Duck.gltf result](https://rawcdn.githack.com/cx20/gltf-test/793866abccb88fe3a70ed7511d818678210a6c4f/examples/filament/index.html?category=sampleModels&model=Duck&scale=1&type=glTF): NG

Is it a matter of usage?

I tried it with Chrome and Firefox on Windows 10 with similar results.

I have unzipped and used the following files.
https://github.com/google/filament/releases/download/v1.9.24/filament-v1.9.24-web.tgz

**Comment by prideout** [comment, marked as answer]
This is fixed with https://github.com/google/filament/pull/3928

**Reply by cx20** [reply]
Thank you for the quick fix. I have confirmed that the problem has been resolved in the latest build.

[Filament v1.9.25rc + Duck.gltf result](https://rawcdn.githack.com/cx20/gltf-test/e92144d6795b97e4c29221e6c3f538a228b27d62/examples/filament/index.html?category=sampleModels&model=Duck&scale=1&type=glTF): OK
